### DISCUSSION TITLE
Devtools / Addons Pro using Gitlab

### DISCUSSION THREAD

**Discussion by abenzakour** [Laravel-Backpack/community-forum#1193] [General] [answered]
Hi, 
We recently purchased Devtools and Addons Pro to test it in a new project, we are a team of developers and we use gitlab to synchronize our work; the problem is when the first developer pushes his work to git; the other developers have a 419 message in all interfaces and not able to use devtools.

**Comment by karandatwani92** [comment]
Hey @abenzakour 

Can you ask your team to ensure that `APP_URL` in the `.env` file is correctly pointing to the URL they use to access the application in the browser, for example: http:127.0.0.1:8000 or http://something.test
```
APP_URL=http://something.test
```

**Reply by abenzakour** [reply]
yes, we are all using the same .env file with http://127.0.0.1:800  
We run `php artisan serve`

**Comment by abenzakour** [comment]
We noticed also some files with 403 error ( Forbidden )

<img width="1474" alt="image" src="https://github.com/user-attachments/assets/1b1cf3a2-a0a3-4687-806b-ad06f84aa94f">

**Comment by abenzakour** [comment, marked as answer]
Resolved by running this command : 

```
cd public
ln -s ../storage/app/public storage
```

**Reply by karandatwani92** [reply]
Ok, that means this step was required for the basset. I'm glad you got it solved. 
```
php artisan storage:link
```
